### PR TITLE
Chore/flush size cfg

### PR DESCRIPTION
**Pull request by mmeloni**

### CODE REVIEW

File: cmd/immudb/immudb.go [hunk 1]
```diff
@@ -51,7 +51,9 @@ Environment variables:
   IMMUDB_AUTH=false
   IMMUDB_PKEY=./tools/mtls/3_application/private/localhost.key.pem
   IMMUDB_CERTIFICATE=./tools/mtls/3_application/certs/localhost.cert.pem
-  IMMUDB_CLIENTCAS=./tools/mtls/2_intermediate/certs/ca-chain.cert.pem`,
+  IMMUDB_CLIENTCAS=./tools/mtls/2_intermediate/certs/ca-chain.cert.pem
+  IMMUDB_EVENTUAL_CONSISTENCY=true
+  IMMUDB_CACHE_SIZE=true`,
```
Comment (padurean) [inline cmd/immudb/immudb.go:56]: @mmeloni shouldn't this be an integer number instead of bool?

File: cmd/immudb/immudb.go [hunk 2]
```diff
@@ -140,6 +146,8 @@ func setupFlags(cmd *cobra.Command, options server.Options, mtlsOptions server.M
 	cmd.Flags().StringP("address", "a", options.Address, "bind address")
 	cmd.Flags().StringP("dbname", "n", options.DbName, "db name")
 	cmd.Flags().StringVar(&o.CfgFn, "config", "", "config file (default path are configs or $HOME. Default filename is immudb.ini)")
+	cmd.Flags().Bool("eventual-consistency", options.EventualConsistency, "enable/disable eventual consistency")
+	cmd.Flags().Uint64("cache-size", options.CacheSize, "enable/disable eventual consistency")
```
Comment (padurean) [inline cmd/immudb/immudb.go:150]: @mmeloni description here looks the same as for the enable/disable config item, i suppose it should be something like _maxcache size in MB_

File: configs/immudb.ini.dist [hunk 3]
```diff
@@ -1,12 +1,14 @@
-dir = .
+dir = ./immudata
 network = tcp
 address = 127.0.0.1
 port = 3322
-dbname = immudata
+dbname = immudb
```
Comment (padurean) [inline configs/immudb.ini.dist:5]: @mmeloni is this ok? i thought it should also be `immudata`

Comment (padurean) [reply configs/immudb.ini.dist:5]: @mmeloni i just saw you've added an intermediary directoy named `immudata`, so i suppose it's ok.

Comment (mmeloni) [reply configs/immudb.ini.dist:5]: Yes the point is that actually dbname is  the name of the db istance. The folder in which data is written is dir. Internally it uses dir+dbname to build the complete path of data folder. @andrei-lihu it's ok?

**Pull request merged**